### PR TITLE
[Review] Request from 'aduffeck' @ 'SUSE/machinery/review_150317_add_verbose_option_to_show'

### PR DESCRIPTION
**Pull request by aduffeck**
Please review the following changes:
  * 06ad287 Add "--verbose" option to "show" command to display used show filters
  * 1efe606 Do not choke on non-existing elements when applying filters

### CODE REVIEW

File: lib/cli.rb [hunk 1]
```diff
@@ -544,6 +544,8 @@ def self.parse_scopes(scope_string)
       desc: "Show diffs of configuration files changes."
     c.switch "html", required: false, negatable: false,
       desc: "Open system description in HTML format in your web browser."
+    c.switch "verbose", required: false, negatable: false,
+      desc: "Show the list of filters that is applied before showing the description."
```
Comment (thardeck) [inline lib/cli.rb]: I would rephrase it to get rid of the singular plural confusion and to keep the same tense as in all other descriptions.
`Show the filters that were applied ...`

**Comment by thardeck** [comment]
lgtm

**Comment by aduffeck** [comment]
Merged.
